pom.xml

```diff
@@ -30,7 +30,7 @@
 
   <properties>
     <stack.version>3.5.0-SNAPSHOT</stack.version>
-    <kafka.version>0.10.2.1</kafka.version>
+    <kafka.version>0.11.0.0</kafka.version>
```

Comment:
we need to agre for this update on vertx-dev list

Reply:
Well since v 10.2. Kafka clients offer backwards compatibility with brokers of older versions so in my opinion it is ok.

Reply:
we also need to check that its transitivie dependencies are fine (logging, Scala version, etc...)

Reply:
have they changed between releases ?

Reply:
it also means that someone having to use 0.10.2.1 will have classloading issues

Reply:
@vietj from what I see from [here](https://github.com/apache/kafka/blob/trunk/gradle/dependencies.gradle) there are minor updates to the transitive dependencies.

Reply:
minor does not mean it does not come without troubles, can you detail which one change ?

Reply:
log4j: 1.2.16 -> 1.2.17 and slf4j 1.7.21 -> 1.7.25.

Reply:
thanks @AGutan we need to check if Kafka can work with the existing dependencies (i.e keep the actual one) and also need to commit to use this new version, it might not be convenient to everyone. @ppatierno what's your opinion on this ?

Reply:
I think that it's quite difficult to check if Kafka can work with the existing dependencies.
@vietj what do you mean with "it also means that someone having to use 0.10.2.1 ... " ? Do you mean having to use the 0.10.2.1 Kafka broker ?